meta/main.yml

```diff
@@ -1,17 +1,19 @@
 ---
 galaxy_info:
-  author: Andrey Shpak
+  license: MIT
+  author: Speechmatics
+  company: Speechmatics
+  namespace: speechmatics
+  role_name: hostname
+  github_branch: add_ci_builds_and_publish_to_galaxy
```

Comment:
To be removed once all hope on publishing as just `speechmatics.hostname` is lost...

Reply:
I finally have a clue as to why this might be happening.

https://github.com/robertdebock/docker-github-action-galaxy/blob/master/Dockerfile doesn't support specifying a Git branch which means by default, metadata/role is imported from `master` branch.

When using `ansible-galaxy` tool directly - it gets it right: https://galaxy.ansible.com/speechmatics/hostname

Which means tagging releases on a feature branch just won't work.


```
$ ansible-galaxy role import --branch add_ci_builds_and_publish_to_galaxy --api-key {REDACTED} --role-name hostname speechmatics ansible-role-hostname -v
No config file found; using defaults
Successfully submitted import request 821730
Starting import: task_id=821730, repository=speechmatics/ansible-role-hostname

===== LOADING ROLE =====
Checking role metadata tags
Checking role platforms
Checking role cloud platforms
Checking role dependencies

===== LINTING ROLE: hostname =====
yamllint OK.
ansible-lint Warnings:
. no-tabs Most files should not contain tabs
. no-tabs Most files should not contain tabs

Updating repository name "ansible_role_hostname" -> "hostname"
===== IMPORTING ROLE: hostname =====

Deleting Content instance: content_type=role, namespace=speechmatics-30690, name=ansible_role_hostname
Updating repository versions...
Galaxy will only import git tags that match the semantic version format, skipping these tag(s): v3.0, v2.1, v2.0, v1.0
Import completed with 3 warnings and 0 errors
```

Perhaps worth an enhancement to the https://github.com/robertdebock/docker-github-action-galaxy  /  https://github.com/robertdebock/galaxy-action